.github/workflows/ci.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pytorch-version: [ '1.8', '1.9', '1.10', '1.11' ]
```

Comment:
do we want to add tests for more recent PyTorches or should that be a separate issue?

Reply:
Oh sure, let me try adding 1.12 and 1.13 (I'm using 1.13.1)